app/react/components/buttons/CopyButton/CopyButton.tsx

```diff
@@ -41,7 +44,7 @@ export function CopyButton({
       <span
         className={clsx(
           copiedSuccessfully && styles.fadeout,
-          styles.displayText,
+          styles.copyButton,
```

Comment:
`copyButton` class doesn't exist in the .css file, but previous `displayText` does